src/test/kotlin/no/nav/familie/ef/søknad/søknad/SøknadsControllerTest.kt

```diff
@@ -0,0 +1,201 @@
+package no.nav.familie.ef.søknad.søknad
+
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import no.nav.familie.ef.søknad.infrastruktur.OppslagSpringRunnerTest
+import no.nav.familie.ef.søknad.infrastruktur.featuretoggle.FeatureToggleService
+import no.nav.familie.ef.søknad.mock.søkerMedDefaultVerdier
+import no.nav.familie.ef.søknad.mock.søknadDto
+import no.nav.familie.ef.søknad.søknad.domain.Kvittering
+import no.nav.familie.ef.søknad.søknad.domain.Person
+import no.nav.familie.ef.søknad.søknad.dto.SøknadBarnetilsynDto
+import no.nav.familie.ef.søknad.søknad.dto.SøknadSkolepengerDto
+import no.nav.familie.kontrakter.felles.objectMapper
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.web.client.exchange
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Primary
+import org.springframework.context.annotation.Profile
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpMethod
+import org.springframework.http.HttpMethod.POST
+import org.springframework.http.HttpStatus
+import org.springframework.test.context.ActiveProfiles
+import java.io.File
+import java.time.LocalDateTime
+
+class SøknadsControllerTest {
+    @Profile("soknads-controller-test")
```

Comment:
```suggestion
    @Profile("soknad-controller-test")
```

Litt småpirk fra min side men da følger man konvensjonen for slik vi prøver å navngi ting og tang. 
Generelt:
Dersom dere kaller det for en `søknadscontroller` så er dette et sammenhengende ord og skal ikke ha camel-case. Dersom dere derimot kaller det for en `søknadController` (uten s) blir det riktig med camel-case. 

Jeg tenker vi kan rename `SøknadsContoller` -> `SøknadController` og `SøknadsContollerTest` -> `SøknadContollerTest` også 🙌